static/js/drafts.js

```diff
@@ -401,10 +410,32 @@ export function launch() {
         return sorted_formatted_drafts;
     }
 
-    function render_widgets(drafts) {
+    function render_widgets(narrow_drafts, other_drafts) {
         $("#drafts_table").empty();
+
+        let narrow_drafts_header;
+        const conversation_header = $t({defaultMessage: "Drafts from this conversation"});
+        const stream_header = $t({defaultMessage: "Drafts from this stream"});
+        // Compose box headers get priority.
+        // TODO: encode this somehow to stay in sync with how filter_drafts_by_compose_box_and_recipient works.
```

Comment:
Yeah, this feels kinda ugly, but I've refactored it a few different ways and am not sure what would look good. If anyone has ideas that would be lovely!